modules/plugin/coverage-multidim/netcdf/src/main/java/org/geotools/coverage/io/netcdf/crs/NetCDFCoordinateReferenceSystemType.java

```diff
@@ -198,6 +186,55 @@ public static NetCDFCoordinateReferenceSystemType parseCRS(CoordinateReferenceSy
         return crsType;
     }
 
+    public static boolean hasConcatenatedTransform(ProjectedCRS projectedCRS) {
```

Comment:
Public and static? Does not seem to require being public. If it needs to, please add javadoc.

Reply:
Right. At the beginning I was using it from another class too. Switching it back to private.